lms/static/sass/views/_shoppingcart.scss

```diff
@@ -651,12 +651,15 @@ $light-border: 1px solid $gray-l5;
         vertical-align: top;
       }
 
-      label {
+      span {
         @extend %t-copy-sub1;
         display: inline-block;
         margin: 0;
+        cursor: text;
         width: ($baseline*60);
         word-wrap: break-word;
+        color: #3c3c3c;
+        font-family: $sans-serif;
```

Comment:
with t-copy-sub1 you don't need to specify font-family, so this line can be removed.